package.json

```diff
@@ -50,7 +50,7 @@
     "core-object": "^1.1.0",
     "ember-cli-babel": "^5.0.0",
     "ember-cli-deploy-plugin": "^0.2.2",
-    "lodash": "^3.9.3",
+    "lodash": "^4.11.1",
```

Comment:
seems stupid but yeah let's revert and maybe do this in another pr
so we can release a beta and get someone to test it just in case.